extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -271,6 +271,7 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     const urlParams = dict({
       'playerId': this.playerId_ || undefined,
       'mediaId': this.mediaId_ || undefined,
+      'url': Services.documentInfoForDoc(element).sourceUrl
```

Comment:
Looks like there's a formatting issue that won't let me merge:
```suggestion
      'url': Services.documentInfoForDoc(element).sourceUrl,
```